apps/dataflow/src/scraper/esim/osm_building_to_json.py

```
@@ -0,0 +1,547 @@
+
+import heapq
+import json
+import math
+import os
+import sys
+import time
+from pathlib import Path
+from typing import Any
+from xml.etree.ElementTree import ParseError
+
+import defusedxml.ElementTree as ET
```

Ruff (0.14.14)

[warning] CamelCase `ElementTree` imported as acronym `ET`

(N817)